src/pages/RepoPage/ComponentsTab/Header/Header.tsx

```diff
@@ -60,15 +61,19 @@ const Header = ({
     <div className="flex flex-col justify-end divide-y divide-solid divide-ds-gray-secondary">
       <div className="grid w-2/3 divide-y divide-solid divide-ds-gray-secondary sm:w-full sm:grid-cols-2 sm:divide-x sm:divide-y-0 md:grid-cols-4">
         <BranchSelector />
-        <div className="mr-4 flex flex-col justify-between gap-2 p-4 sm:border-l sm:border-ds-gray-secondary sm:py-0 md:border-l-0">
+        <div className="md:w-[16rem]">
```

Comment:
![Screenshot 2024-04-22 at 3 10 44 PM](https://github.com/codecov/gazebo/assets/148245014/231e9b46-6117-49c1-8b81-82bca7fb7d8d)

Looks like the number disappears when we have a small - medium screen size. Let's either fix it or create a ticket to follow up.